driver-async/src/main/com/mongodb/async/client/gridfs/GridFSUploadStream.java

```diff
@@ -33,7 +34,14 @@
      *
      * @return the ObjectId for the file to be uploaded
      */
-    ObjectId getFileId();
+    ObjectId getObjectId();
```

Comment:
Update the javadoc to indicate that that can throw an exception if a custom file id that's not an objectid was provided